<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Docker_mover</class>
 <widget class="QDialog" name="Docker_mover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>125</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QToolButton" name="btn_up">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../common/lib/pictures/arrow.qrc">
         <normaloff>:/arrow/arrow-up.png</normaloff>:/arrow/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="btn_down">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../common/lib/pictures/arrow.qrc">
         <normaloff>:/arrow/arrow-down.png</normaloff>:/arrow/arrow-down.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btn_left">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../common/lib/pictures/arrow.qrc">
         <normaloff>:/arrow/arrow-left.png</normaloff>:/arrow/arrow-left.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btn_right">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../../../common/lib/pictures/arrow.qrc">
         <normaloff>:/arrow/arrow-right.png</normaloff>:/arrow/arrow-right.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_up</tabstop>
  <tabstop>btn_left</tabstop>
  <tabstop>btn_right</tabstop>
  <tabstop>btn_down</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../../../../common/lib/pictures/arrow.qrc"/>
 </resources>
 <connections/>
</ui>
